native/src/utils/renderHtml.ts

```diff
@@ -394,6 +394,34 @@ const renderHtml = (
         font-weight: bold;
       }
 
+      .contact-card {
+        display: inline-block;
+        box-sizing: border-box;
+        padding: 16px;
+        border-radius: 4px;
+        background-repeat: no-repeat;
+        background-color: rgb(127 127 127 / 15%);
+        background-image: linear-gradient(to right, rgb(255 255 255 / 90%) 0 100%),
+          url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNMTMuMDE4IDEyLjQ4aC0yLjAzNkE5LjA5IDkuMDkgMCAwIDAgMS45MiAyMS42YS40OC40OCAwIDAgMCAuNDguNDhoMTkuMmEuNTMuNTMgMCAwIDAgLjQ4LS41MzggOS4wOCA5LjA4IDAgMCAwLTkuMDYyLTkuMDYyTTE2LjggNi43MmE0LjggNC44IDAgMCAxLTQuOCA0LjggNC44IDQuOCAwIDAgMS00LjgtNC44IDQuOCA0LjggMCAwIDEgNC44LTQuOCA0LjggNC44IDAgMCAxIDQuOCA0LjgiLz48L3N2Zz4=');
+        background-blend-mode: difference;
+        background-position: calc(100% + 32px) calc(100% + 16px);
+        background-size: 108px;
+        box-shadow: 0 1px 1px rgb(0 0 0 / 40%);
+        width: 100%;
+
+        p {
+          margin-top: 4px;
+        }
+
+        h4 {
+          margin: 0 0 12px;
```

Comment:
```suggestion
          margin-top: 0;
          margin-bottom: 12px;
```
I for one can never remember what the shortcut with three numbers means 😅